RULES.md

```diff
@@ -0,0 +1,94 @@
+# Traffic rules
+
+## TLDR
+We need to:
+
+Allow independent groups in the content project to work at their own pace on parts of the project that don’t influence each other.
+Introduce enter and exit criteria for roles with elevated privileges so the hierarchy can self-organize.
+Define a framework for organizations to join the project and assume responsibilities.
+Make sure that the project stays coherent and individual parts contribute to its value, and the project doesn’t become a disgusting spaghetti soup.
+
+
+## Decomposition
+
+In order to keep the project scalable, we divide the project into separate areas, and assign groups of users that can be granted commit access with the intention to develop content in respective areas.
+Those areas are going to be segmented by
+
+- Products (RHEL, Ubuntu, …) - make it easy to develop content of a product that doesn’t influence the rest of the build system.
+- Product-specific profiles (PSPs s.a. Firefox STIG, Ubuntu CIS, …) and respective control files - make it easy for SMEs that only want to assign rules to profiles without going into details.
+- Product-independent profiles (PIPs s.a. ANSSI, HIPAA, PCI-DSS, …) and respective control files - although PIP development that benefits the whole community instead of cluttering the content with “if product in [...]”.
+- Shared resources - rules only loosely coupled to products, templates, CPEs etc.
+- Build system - decisions upon architecture of the build system impact build time, project capabilities, and can also move maintenance costs.
+- Test-related code - ensure that tests have the greatest coverage as possible, but that don’t waste time and don’t suffer from false positives.
+- Organization administration - ability to grant and remove rights, create and delete repositories etc.
+
+
+## Implementation
+
+### Products
+
+- Keeping track of the decomposition: Metadata in product.yml files, CODEOWNERS file
+- Area of effect: All PSPs that don’t have specific stakeholders, associated content and test scenarios.
```

Comment:
It looks like my question above about additive vs. net is described here, but I'm not quite sure if that can be reflected in a CODEOWNERS file well? Maybe I'm wrong, its been a while since I looked at that, so perhaps the suggestion of a RH example would be good here.

Are test scenarios something handled automatically (e.g., ssgts and are we going to be running that across products on releases)? Or documented in the project? Or just internally at specific orgs?

Reply:
I don't understand what exactly you mean by additive vs net or what is the "handling" of test scenarios.